geode-core/src/test/java/org/apache/geode/management/internal/operation/OperationHistoryManagerTest.java

```diff
@@ -103,9 +104,33 @@ public void getSomeCompleted() {
     assertThat(history.getStatus("3")).isNotNull();
   }
 
+  @Test
+  public void timestampsAreCorrect() throws Exception {
```

Comment:
split this up into three methods. Instead of using "now = System.currentTImeMillis()" use the "new Date()" the test passed in when op() was called. Maybe you should just add a Date parameter to op(String, CompletableFuture)

Reply:
split into multiple tests and removed use of `now`, instead passing start date into constructor